oak_functions_containers_container/build_container_bundle

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# A simple script to create a tarball of an example OCI Runtime Bundle (https://github.com/opencontainers/runtime-spec/blob/4e3b9264a330d094b0386c3703c5f379119711e8/bundle.md)
+# that can be executed on Oak Containers. Used for manual testing during
+# development. Down the line we probably want to replace this.
+
+set -o errexit
```

Comment:
Please set all the safe options here, like

https://github.com/project-oak/oak/blob/8300e9945f4397d4b29aa23d8e925da7dc451b30/scripts/common#L4-L7

Reply:
done.